css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  font-family: "Brandon Grotesque", sans-serif;
+}
+
+body {
+  margin-top: 40px;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+}
+
+h1 {
+  color: #ec5d57;
+  font-size: 20px;
+  font-weight: 600;
+  text-align: center;
+}
+
+form {
+  max-width: 470px;
+  margin: auto;
+}
+
+label {
+  font-size: 12px;
+  font-weight: 600;
+  color: #2f2f2f;
+  margin-left: 15px;
+}
+
+::placeholder {
+  font-family: AtlasGrotesk, sans-serif;
+  font-size: 16px;
+  font-weight: 500;
+  color: #a6aaa9;
+}
+
+input,
+select {
+  width: 470px;
+  height: 40px;
+  border-color: #a6aaa9;
+  border-width: 1px;
+}
+
+input {
+  padding-left: 10px;
+  border-radius: 8px;
+}
+
+select {
+  background-color: #f3f3f3;
+  color: #2f2f2f;
+  font-size: 16px;
+  font-weight: 500;
+  padding-left: 5px;
+  border-radius: 5px;
+}
+
+button {
+  background-color: #ec5d57;
+  color: white;
+  font-size: 14px;
+  font-weight: 500;
+  text-transform: uppercase;
+  width: 90px;
+  height: 35px;
+  border-width: 0;
+  border-radius: 20px;
+  margin-top: 10px;
+  margin-left: 380px;
```

Comment:
WOAH! That's a lot of margin! I'm assuming you are using this to push the element to the right side of the form, yes? Margin should only be used for minor adjustments on elements, not as a layout tool. For something like this, you could simply set the `div` element that is the parent of the `button` to have `text-align` set to `right`, and the button will shift to the right side of the container.

https://developer.mozilla.org/en-US/docs/Web/CSS/text-align

Reply:
Ahh so that's how you do it, gotcha!! 😃